metadata.yaml

```diff
@@ -47,6 +47,9 @@ provides:
   cni:
     interface: kubernetes-cni
     scope: container
+  container-runtime:
+    interface: container-runtime
+    scope: container
```

Comment:
No need for this since it will be brought in with https://github.com/charmed-kubernetes/layer-kubernetes-master-worker-base/pull/1 right?